scenarios/traffic_anomaly_detection/daily_rollup.sql

```diff
@@ -0,0 +1 @@
+SELECT TD_DATE_TRUNC('day', time) AS day, td_host, COUNT(1) AS daily_pv FROM ${source_table} WHERE TD_TIME_RANGE(time, TD_TIME_ADD(TD_DATE_TRUNC('day', TD_SCHEDULED_TIME()), '-60d')) AND LOWER(td_user_agent) NOT LIKE '%bot%' GROUP BY 1,2 ORDER BY 1
```

Comment:
`LOWER(td_user_agent) NOT LIKE '%bot%'`
td_user_agent is based on an older user agent parser since there is a performance issue.
https://treasure-data.atlassian.net/browse/PLT-8422

Due to it, we need to use PARSE_USER_AGENT instead of the column.

```
SELECT 
  TD_DATE_TRUNC(
    'day',
    time
  ) AS DAY,
  td_host,
  COUNT(1) AS daily_pv
FROM
  ${source_table}
WHERE
  TD_TIME_RANGE(time,
    TD_TIME_ADD(TD_DATE_TRUNC('day',
        TD_SCHEDULED_TIME()),
      '-60d'))
  AND element_at(TD_PARSE_AGENT(agent), 'category') NOT IN ('crawler', 'misc', 'unknown')
GROUP BY
  1,
  2
ORDER BY
  1
```